packages/fluentui/docs/src/examples/components/Toolbar/Performance/CustomToolbar.perf.tsx

```diff
@@ -337,27 +337,29 @@ interface CustomToolbarProps {
 
 type CustomToolbarLayout = (
   props: CustomToolbarProps,
-) => ShorthandCollection<ToolbarItemProps | ToolbarCustomItemProps, ToolbarItemShorthandKinds>;
+) =>
+  | ShorthandCollection<ToolbarItemProps | ToolbarCustomItemProps, ToolbarItemShorthandKinds>
+  | ShorthandCollection<ToolbarItemProps | ToolbarCustomItemProps>;
```

Comment:
```suggestion
) => ToolbarProps['items']
```

I think this should work

Reply:
It didn't really worked